pkg/ingester/errors.go

```diff
@@ -430,3 +441,20 @@ func newIngesterErrSamplers(freq int64) ingesterErrSamplers {
 		log.NewSampler(freq),
 	}
 }
+
+func handlePushError(err error) error {
```

Comment:
Note for reviewers: this method has been moved from `ingester.go` to `errors.go`.